### PR TITLE
Bouncing: add support for all directions

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
Two commits:
 * Bouncing: add support for all directions
 * ArcListview: use verticalAndHorizontal bouncing

### CODE REVIEW

File: src/js/core/widget/core/scroller/effect/Bouncing.js [hunk 1]
```diff
@@ -61,7 +62,8 @@
 				},
 				Orientation = {
 					VERTICAL: "vertical",
-					HORIZONTAL: "horizontal"
+					HORIZONTAL: "horizontal",
+					VERTICALANDHORIZONTAL: "verticalAndHorizontal"
```
Comment (TomaszLukawskiSam) [inline src/js/core/widget/core/scroller/effect/Bouncing.js]: This property name makes some confusion... maybe: VERTICAL_HORIZONTAL

Comment (grzegorz-czajkowski) [reply src/js/core/widget/core/scroller/effect/Bouncing.js]: Changed to VERTICAL_HORIZONTAL
Plus changed property value to: vertical-horizontal

**Comment by grzegorz-czajkowski** [comment]
@TomaszLukawskiSam I did refactoring as we discussed. Can you have a look?

**Comment by TomaszLukawskiSam** [comment]
> @TomaszLukawskiSam I did refactoring as we discussed. Can you have a look?

Try to solve CI issues before.

**Comment by grzegorz-czajkowski** [comment]
There is a hang while reaching the following test line:
`                        ok(this.scrollbars[0].classList.contains("ui-show"), "Left scrollbar bouncing is visibile");
`
this.scrollbars is NodeList object whose length is 2. I need to investigate it more.

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**